LICENSE.md

```diff
@@ -1,8 +1,6 @@
-CC0 1.0 Universal
-==================
+# CC0 1.0 Universal
```

Comment:
Similarly, this one.

Reply:
It's just the "Markdown" extension on visual studio code.